pkgs/development/compilers/ecl/default.nix

```diff
@@ -36,38 +36,12 @@ stdenv.mkDerivation {
   };
 
   patches = [
-    # https://trac.sagemath.org/ticket/22191#comment:237
-    (fetchurl {
-      name = "ECL_WITH_LISP_FPE.patch";
-      url = "https://git.sagemath.org/sage.git/plain/build/pkgs/ecl/patches/ECL_WITH_LISP_FPE.patch?h=9.2";
-      sha256 = "0b194613avcmzr1k9gq725z41wdkg5rsa0q21kdw050iqpprcj1c";
-    })
-
     # https://gitlab.com/embeddable-common-lisp/ecl/-/merge_requests/1
-    (fetchurl {
+    (fetchpatch {
       name = "write_error.patch";
       url = "https://git.sagemath.org/sage.git/plain/build/pkgs/ecl/patches/write_error.patch?h=9.2";
       sha256 = "1lvdvh77blnxp0zbd27dsbq1ljkb5qblabf1illszn4j7qgq88fh";
```

Comment:
I think you need to check if the hash changed. It most likely did.

Reply:
Good call, fixed now, thanks.